### PR TITLE
Feature/bphh 571/coord lookup rebase

### PR DESCRIPTION
**Pull request by sunto**
## Description

Adds ability to lookup addresses based on pids, ability to find things based on pins

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📦 Chore (Release)
- [ ] ✅ Test

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-515
https://hous-bssb.atlassian.net/browse/BPHH-1545

## Steps to QA

Backend for developers to test new functions against LTSA and Geocoder Wrapper.
On the select of a new PIN, type 10 Semana Crescent.  It should give you an address, you select it, it has no PID.  You can try a PIN, it will try to validate it.  It will show if it cannot verify it.  If you delete the PIN it should clear out.

@derekyau - please review the new ruby gems, they require local installs / binaries for geo location.  (open the gem documentation)

### CODE REVIEW

**Comment by bkmaibach** [comment]
Creating a permit application with a known North Van address now produces:

image

**Comment by bkmaibach** [comment]
We need to do this before this can be merged:

https://hous-bssb.atlassian.net/browse/BPHH-1617

Also the code you wrote doesn't seem to meet the criteria of the listed card
https://hous-bssb.atlassian.net/browse/BPHH-515 (pid to address lookup) unless I am missing something

File: Gemfile [hunk 1]
```diff
@@ -78,6 +78,12 @@ gem "simple-feed", "~> 3.1.2"
 gem "rswag-api", "~> 2.13"
 gem "rswag-ui", "~> 2.13"
 
+# brew install geos
+# bundle config build.rgeo --with-geos-dir=/usr/local/opt/geos
+gem "rgeo"
```
Comment (derekyau) [inline Gemfile:83]: lock these gem versions down

**Comment by bkmaibach** [comment]
@sunto this feature is still giving errors. entering a PID does not populate address and gives this

image

**Comment by bkmaibach** [comment]
@sunto still getting this error:

image

**Review by bkmaibach — APPROVED** [review]
Tested working

**Pull request merged**